src/modules/events/send_event.h

```diff
@@ -51,52 +51,97 @@ extern "C" __EXPORT int send_event_main(int argc, char *argv[]);
+	 * @return Returns 0 iff successful, otherwise < 0 on error.
+	 */
 	static int custom_command(int argc, char *argv[]);
 
-	/** @see ModuleBase */
+	/**
+	 * @see ModuleBase
+	 * @brief Prints usage options to the console.
+	 * @param reason Pointer to... (something important? Perhaps the number 42?... :) )
+	 *               @todo: Assitance required to fill in intent for the param 'reason'.
+	 * @return Returns 0 iff successful, otherwise < 0 on error.
+	 */
 	static int print_usage(const char *reason = nullptr);
 
-private:
+	/**
+	 * @brief Initialize class in the same context as the work queue and start the background listener.
+	 * @param argc The task argument count.
+	 * @param argc Pointer to the task argument variable array.
+	 * @return Returns 0 iff successful, otherwise < 0 on error.
+	 */
+	static int task_spawn(int argc, char *argv[]);
 
-	/** Start background listening for commands
-	 *
-	 * @return 0 if successful, <0 on error. */
-	int start();
+private:
 
+	/**
+	 * @brief Returns an ACK to a vehicle_command.
+	 * @param cmd The vehicle command struct being referenced.
+	 * @param result The command acknowledgement result.
+	 */
+	void answer_command(const vehicle_command_s &cmd, unsigned result);
 
-	/** Trampoline for initialisation. */
-	static void initialize_trampoline(void *arg);
-	/** Trampoline for the work queue. */
+	/**
+	 * @brief Trampoline for the work queue.
+	 * @param arg Pointer to the task startup arguments.
+	 */
 	static void cycle_trampoline(void *arg);
 
-	/** call process_commands() and schedule the next cycle. */
+	/**
+	 * @brief Calls process_commands() and schedules the next cycle.
+	 */
 	void cycle();
 
-	/** check for new commands and process them. */
+	/**
+	 * @brief Trampoline for initialisation.
+	 * @param arg Pointer to the task startup arguments.
+	 */
+	static void initialize_trampoline(void *arg);
+
+	/**
+	 * @brief Checks for new commands and processes them.
+	 */
 	void process_commands();
 
-	/** return an ACK to a vehicle_command */
-	void answer_command(const vehicle_command_s &cmd, unsigned result);
+	/**
+	 * @brief Starts background task listening for commands.
+	 * @return Returns 0 iff successful, otherwise < 0 on error.
+	 */
+	int start();
 
+	/** @struct _work The wok queue struct. */
 	static struct work_s _work;
+
+	/** @param _subscriber_handler The uORB subscriber handler. */
 	SubscriberHandler _subscriber_handler;
+
+	/** @param _status_display Pointer to the status display uORB topic. */
 	status::StatusDisplay *_status_display = nullptr;
+
+	/** @param _rc_loss_alarm Pointer to the RC loss alarm uORB topic. */
 	rc_loss::RC_Loss_Alarm *_rc_loss_alarm = nullptr;
+
+	/** @param _command_ack_pub The command ackowledgement topic. */
 	orb_advert_t _command_ack_pub = nullptr;
 
+	/** @note Declare local parameters using defined parameters. */
 	DEFINE_PARAMETERS(
+		/** @param _param_status_display The LED status display. */
```

Comment:
A better comment would be: `Parameter to enable/disable the LED status display` (same below)

Reply:
Updated as such.  Thanks!